plugin/evm/atomic_backend.go

```diff
@@ -96,6 +108,14 @@ func NewAtomicBackend(
 	if err != nil {
 		return nil, err
 	}
+	if len(bonusBlocksRlp) > 0 {
+		if err := atomicTrie.repairAtomicTrie(bonusBlocks, bonusBlocksRlp); err != nil {
+			return nil, err
+		}
+		if err := db.Commit(); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
After repairing the atomic trie, should we free the memory associated with `bonusBlocks`? May be overkill for its size.

Reply:
added code to free it in vm.Initialize()